src/v2/Apps/Show/Components/ShowEvents.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowEvents_show } from "v2/__generated__/ShowEvents_show.graphql"
+
+interface ShowEventsProps {
+  show: ShowEvents_show
+}
+
+export const ShowEvents: React.FC<ShowEventsProps> = ({ show }) => {
+  const events = show.events
+
+  if (!events || events.length === 0) {
+    return null
+  }
+
+  //TODO: flesh out show event details once we have more to QA
+  //TODO: include render test??
+
+  return (
+    <Box>
+      {events && (
+        <>
+          <Text variant="xs" as="h3" mb={1}>
+            Show Events
+          </Text>
+
+          {events.map(event => {
+            return (
+              <Box key={show.internalID}>
+                <Text>{event?.eventType}</Text>
+              </Box>
+            )
+          })}
+        </>
+      )}
+    </Box>
+  )
+}
+
+export const ShowEventsFragmentContainer = createFragmentContainer(ShowEvents, {
+  show: graphql`
+    fragment ShowEvents_show on Show {
+      events {
+        title
+        eventType
+        description
+        startAt
+        endAt
```

Comment:
@TMcMeans - note that these fields should be configurable and return any format that you'd like. See [here](https://github.com/artsy/force/blob/15693e3b173c8daa510d72014eb0bf378a6eacd7/src/v2/Apps/Home/Components/HomeCurrentFairs.tsx#L179-L180) for an example. 

Also, check out this list of date formats: https://docs.oracle.com/cd/E41183_01/DR/Date_Format_Types.html#:~:text=MM%2FDD%2FYY,(17%2F02%2F2009)

Reply:
oh awesome, that solves the problem of having to do any data cleanup for sure thanks!

Reply:
There's a few fields in metaphysics like that. Another example is [`description`](https://github.com/artsy/force/blob/main/src/v2/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromArtsy.tsx#L70). And here's it defined in [Metaphysics](https://github.com/artsy/metaphysics/blob/main/src/schema/v2/gene.ts#L125). Lil helpers that will transform inputs into new fields that one can configure based on the need.